foundation/eclipselink.extension.corba.test/antbuild.xml

```diff
@@ -0,0 +1,674 @@
+         <!-- Determine availability of other dependencies. -->
+        <condition property="dependency.ok">
+            <and>
+                <isset property="test.framework.exist"/>
+                <isset property="core.test.models.exist"/>
+                <isset property="junit.lib.exist"/>
+                <isset property="jdbc.jar.exist"/>
+            </and>
+        </condition>
+        <!-- Define properties for corba -->
+        <property name="test.run.corba.server.port" value="1122"/>
+        <property name="test.run.naming.factory" value="com.sun.enterprise.naming.SerialInitContextFactory"/>
+        <!-- Define system properties for resource processing and testing  -->
+        <property name="eclipselink.logging.level" value="${logging.level}"/>
+        <propertyset id="test.run.propertyset" dynamic="false">
+            <propertyref prefix="test.security-manager"/>
+            <propertyref prefix="test.run"/>
+            <propertyref regex="db\.(?!pwd|url|oci|sys).*"/>
+            <!-- oradb specific properties -->
+            <propertyref regex="^pa\..*[^d]$"/>
+            <propertyref name="jdbc.driver.jar"/>
+            <propertyref name="eclipselink.logging.level"/>
+        </propertyset>
+        <propertyset id="test.perfrun.propertyset" dynamic="false">
+            <propertyref regex="eclipselink.perfdb\.(?!pwd).*"/>
+        </propertyset>
+    </target>
+
+    <target name="download-gf-corba">
+        <mkdir dir="${corba_test.run.dir}"/>
+        <mkdir dir="${corba_test.run.dir}/gf-corba"/>
+        <condition property="M2_HOME" value="${env.M2_HOME}" else="~/apache-maven-3.5.2">
+            <isset property="env.M2_HOME"/>
+        </condition>
+        <java dir="${corba_test.run.dir}" fork="true" failonerror="true" classname="org.codehaus.plexus.classworlds.launcher.Launcher">
+            <jvmarg value="-Dclassworlds.conf=${M2_HOME}/bin/m2.conf"/>
+            <jvmarg value="-Dmaven.home=${M2_HOME}"/>
+            <jvmarg value="-Dmaven.multiModuleProjectDirectory=${corba_test.run.dir}"/>
+            <jvmarg value="-Dartifact=org.glassfish.main.appclient:gf-client:5.1.0"/>
+            <arg value="dependency:get"/>
+            <classpath>
+                <fileset dir="${M2_HOME}/boot">
+                    <include name="plexus-classworlds-*.jar"/>
+                </fileset>
+            </classpath>
+        </java>
+
+        <java dir="${corba_test.run.dir}" fork="true" failonerror="true" classname="org.codehaus.plexus.classworlds.launcher.Launcher">
+            <jvmarg value="-Dclassworlds.conf=${M2_HOME}/bin/m2.conf"/>
+            <jvmarg value="-Dmaven.home=${M2_HOME}"/>
+            <jvmarg value="-Dmaven.multiModuleProjectDirectory=${corba_test.run.dir}"/>
+            <jvmarg value="-DoutputDirectory=${corba_test.run.dir}/gf-corba"/>
+            <arg line="-f ${user.home}/.m2/repository/org/glassfish/main/appclient/gf-client/5.1.0/gf-client-5.1.0.pom"/>
```

Comment:
Hard coded reference to Maven repository. If Maven repo will be located in other directory build will fail. See _utils/eclipselink.dbws.builder.test/antbuild.xml:135,143_ too.

Reply:
right, this should be resolved migration to maven. Currently there is no simple way to add this to compdeps features since jaxws-rt does not contain OSGi headers. If this causes a problem on the build infra, I'll solve it in an extra PR

Reply:
May be configure it instead of Ant scripts into _buildsystem/compdeps/org.eclipse.persistence.compdeps.p2repo/pom.xml_ as a part of maven-dependency-plugin configuration.

Reply:
allow override through property